src/Symfony/Component/HttpClient/Response/TraceableResponse.php

```diff
@@ -132,6 +170,9 @@ public static function stream(HttpClientInterface $client, iterable $responses,
         }
 
         foreach ($client->stream($wrappedResponses, $timeout) as $r => $chunk) {
+            if ($traceableMap[$r]->event && $traceableMap[$r]->event->isStarted() && $chunk->isLast()) {
```

Comment:
calling is last without checking for getError will throw when an error happens (same for timeouts)

Reply:
Good catch.
I added `lap` on timeout and chunk (same for `getStatus` and `getHeader`) To show a "tick" when content is partially received.

![Screenshot from 2020-10-23 10-52-45](https://user-images.githubusercontent.com/578547/96977841-eaa74e00-151d-11eb-939e-68717cec22ec.png)

```php
        $response1 = $client->request('GET', 'https://httpstat.us/200');
        $response1->getContent(false);
        $response2 = $client->request('GET', 'https://httpstat.us/404');
        $response2->getContent(false);

        $responses[] = $client->request('GET', 'https://httpstat.us/200?sleep=3000');
        $responses[] = $client->request('GET', 'https://httpstat.us/200?sleep=500');
        try {
            foreach ($client->stream($responses, 1) as $chunk) {}
        } catch (\Exception $e)
        {}
        sleep(1);
        foreach ($responses as $response) {
            $response->getContent();
        }
```